src/main/java/com/adobe/aem/compgenerator/utils/CommonUtils.java

```diff
@@ -245,6 +246,7 @@ private static BufferedWriter getFileWriterFromTemplate(String path,
             map.put("sightly", StringUtils.uncapitalize(generationConfig.getJavaFormatedName()));
             map.put("slingModel", generationConfig.getProjectSettings().getModelInterfacePackage() + "." + generationConfig.getJavaFormatedName());
             map.put("CODEOWNER", generationConfig.getProjectSettings().getCodeOwner());
+            map.put("htmlOutput", generationConfig.getOptions().isHtmlContent() ? HTMLUtils.renderHtml(generationConfig) : "<!-- Component HTML goes here -->");
```

Comment:
Let's add the 4 spaces before `<!-- Component HTML goes here -->` to get the correct indent, consistent with how it was before this PR.

Reply:
Added